apps/designer/app/designer/designer.tsx

```diff
@@ -114,6 +114,13 @@ const useCopyPaste = (publish: Publish) => {
   });
 };
 
+const useSetWindowTitle = () => {
+  const [project] = useProject();
+  useEffect(() => {
+    document.title = `${project?.title} | Webstudio`;
+  }, [project?.title]);
+};
```

Comment:
Why can't we use the standard Remix way to set title?

Reply:
Ah, it's because it can be updated locally after the page is loaded?

Reply:
Standard way would be using serverside meta function, which doesn't have access to project and I wanted to have project title there because its confusing with multiple tabs open.

We only get project when we render the designer ui